concordium-sdk/src/main/java/com/concordium/sdk/types/AbstractAddress.java

```diff
@@ -31,26 +28,6 @@ public abstract class AbstractAddress {
         this.type = type;
     }
 
-    //not pretty - find a better way of handling this.
-    public static AbstractAddress parseAccount(Map<String, Object> o) {
```

Comment:
Did you find a better way, or was this not needed anyway?

Reply:
This was unused and not needed anymore. It was used for parsing accounts via the old grpc API.